### PR TITLE
32 component modeling

### LINKED ISSUE #32: Home-Page Component Modeling
This Issue is for updating the component-model, filter and definition for home-page blocks.

### PR DESCRIPTION
**Pull request by shivangi1422**
Component Modelling changes for Card, Mini-Teasers and Alternating blocks.

Fix #32 

Test URLs:
- Before: https://main--danaher-abcam--aemsites.aem.live/en-us
- After: https://32-component-modeling--danaher-abcam--aemsites.aem.live/en-us

### CODE REVIEW

**Review by pardeepgera23 — APPROVED** [review]

File: component-definition.json [hunk 1]
```diff
@@ -148,6 +148,51 @@
...
           },
+          {
+            "title": "Mini-Teasers",
+            "id": "mini-teasers",
+            "plugins": {
+              "xwalk": {
+                "page": {
+                  "resourceType": "core/franklin/components/block/v1/block",
+                  "template": {
+                    "name": "Mini-Teasers",
```
Comment (rgravitvl) [inline component-definition.json:159]: Model also to be added here as : "model": ""mini-teasers",

File: component-models.json [hunk 2]
```diff
@@ -135,16 +135,105 @@
...
+          "name": "card_CTAhrefText",
+          "value": "",
+          "label": "CTA Text"
+        },
+        {
+          "component": "text",
+          "valueType": "string",
+          "name": "card_CTAhrefTitle",
+          "value": "",
+          "label": "CTA Title"
```
Comment (rgravitvl) [inline component-models.json:215]: How does differentiate from CTA Text? I see both are component type as text

File: component-models.json [hunk 3]
```diff
@@ -219,6 +308,180 @@
...
+            {
+              "name": "h3",
+              "value": "h3"
+            },
+            {
+              "name": "h4",
+              "value": "h4"
+            },
+            {
+              "name": "h5",
```
Comment (rgravitvl) [inline component-models.json:425]: Do we really need all these heading types??

**Review by rgravitvl — CHANGES_REQUESTED** [review]
@shivangi1422  We have review the model JSON, can we connect tomorrow morning 10:30am??

**Review by pardeepgera23 — APPROVED** [review]

**Review by rgravitvl — APPROVED** [review]

**Pull request merged**